tests/phpunit/tests/user.php

```diff
@@ -1127,11 +1127,24 @@ public function test_search_users_url() {
 		$this->assertContains( (string) self::$contrib_id, $users );
 	}
 
+	public function test_search_users_no_email() {
```

Comment:
This test doesn't seem necessary to me. We don't have inversions of all the other search fields.